fuzzer_http_processor/README.md

```diff
@@ -0,0 +1,4 @@
+Fuzzer HTTP Processor scripts
```

Comment:
The name of the directory should match the one in the templates dir (i.e. "httpfuzzerprocessor"), so ZAP properly loads the scripts.